mlflow/gateway/providers/gemini.py

```diff
@@ -38,6 +38,15 @@
 
 
 class GeminiAdapter(ProviderAdapter):
+    @classmethod
+    def _normalize_finish_reason(cls, finish_reason):
```

Comment:
sorry for the drive-by change, but I found this bug during implementing this PR.